packages/server/package.json

```diff
@@ -63,6 +63,7 @@
     "@socket.io/redis-adapter": "^8.2.1",
     "airtable": "0.10.1",
     "arangojs": "7.2.0",
+    "archiver": "^7.0.1",
```

Comment:
Should we pin this library version? I know there are some that aren't like the `redis-adapter` above - but we probably should keep pinning things for clarity/safety.